lib/danger/dangerfile.rb

```diff
@@ -35,13 +35,13 @@ def print_known_info
 
         rows << [key.to_s, value]
       end
-
+      rows << ["PR Body", pr_body.scan(/.{80}/)]
```

Comment:
I was hoping I could chuck in an array as lines and it would wrap, but it looks like it will just try and go on forever. Agree, switching to 80 char preview.